app/templates/players/index.hbs

```diff
@@ -25,4 +25,23 @@
     <button {{on "click" (fn this.onFilterClick)}}>Filter</button>
 </label>
 
-<Players::List @players={{model}} @onHeaderClick={{this.onHeaderClick}} @sortDirection={{this.sortDirection}} />
+<Players::List
+    @players={{model}}
+
+    @onHeaderClick={{this.onHeaderClick}}
+    @sortDirection={{this.sortDirection}}
+
```

Comment:
no need to leave empty lines here

Reply:
Add https://prettier.io/ script to the package.json for formatting.